src/rdb/rdb.c

```diff
@@ -80,7 +80,8 @@ rdb_create(const char *path, const uuid_t uuid, size_t size,
 	 * rdb_start() checks this attribute when starting a DB.
 	 */
 	d_iov_set(&value, (void *)uuid, sizeof(uuid_t));
-	rc = rdb_mc_update(mc, RDB_MC_ATTRS, 1 /* n */, &rdb_mc_uuid, &value);
+	rc = rdb_mc_update(mc, RDB_MC_ATTRS, true /* crit */, 1 /* n */,
+			   &rdb_mc_uuid, &value);
```

Comment:
In an early version of the experiment, as a "big hammer", I had made all MC updates specify the flag to vos_ob_update() and all LC updates not specify the flag (without changing the any of the rdb_mc_update and rdb_lc_update function signatures).

Yes, I think it may be worth leaving rdb_mc_update() untouched (always specify VOS_OF_CRIT), and having rdb_lc_update() accept the new boolean argument.

Reply:
I see. One less parameter would be nice---I've been painfully counting for the n-th parameter of dtx_begin a lot when rebasing some of my PRs recently...